services/agent/src/simcore_service_agent/modules/volumes_cleanup/_core.py

```diff
@@ -43,13 +43,7 @@ async def backup_and_remove_volumes(settings: ApplicationSettings) -> None:
                 logger.error("%s", e)
                 continue
 
-            logger.info(
-                (
-                    "Succesfully pushed data to S3 for zombie dynamic sidecar "
-                    "docker volume: '%s'"
-                ),
-                volume_name,
-            )
+            logger.info(("Successfully cleaned up docker volume: '%s'"), volume_name)
```

Comment:
there i no need for the parentrheis in the first argument

Reply:
will remove the extra parentheses